rosdoc2/verbs/build/builders/sphinx_builder.py

```diff
@@ -609,10 +609,18 @@ def generate_wrapping_rosdoc2_sphinx_project_into_directory(
                     os.path.abspath(user_sourcedir),
                     os.path.abspath(directory),
                     dirs_exist_ok=True)
-                shutil.copytree(
-                    os.path.abspath(package_src_directory),
-                    os.path.abspath(directory),
-                    dirs_exist_ok=True)
+                if (self.build_context.build_type == 'ament_python'):
```

Comment:
Nit: no need for parentheses in python:

```suggestion
                if self.build_context.build_type == 'ament_python':
```